packages/code-assist/README.md

```diff
@@ -330,6 +330,150 @@ This server supports two standard MCP communication protocols:
 
 -----
 
+<!-- [START maps_StreamableHTTPGuide] -->
+
+## 🌐 StreamableHTTP Developer Guide
+
+For developers who need to integrate the Google Maps Platform Code Assist MCP server using the StreamableHTTP transport, this guide provides detailed setup instructions, configuration examples, and troubleshooting tips.
+
+### When to Use StreamableHTTP
+
+Use StreamableHTTP transport when:
+- Your client requires a URL-based connection instead of command execution
+- You need to run the server remotely (e.g., on Google Cloud Run)
+- Your development environment requires HTTP-based communication
+- You're building custom integrations that communicate over HTTP
+
+### Local StreamableHTTP Setup
+
+**1. Start the Server**
+
+Run the MCP server locally with a specific port:
+
+```bash
+npm start -- --port 3215
+```
+
+Or using npx:
+
+```bash
+npx -y @googlemaps/code-assist-mcp@latest --port 3215
+```
+
+The server will be available at: `http://localhost:3215/mcp`
+
+**2. Client Configuration**
+
+Configure your MCP client to use the StreamableHTTP endpoint:
+
+```json
+{
+  "mcpServers": {
+    "google-maps-platform-code-assist": {
+      "url": "http://localhost:3215/mcp",
+      "type": "streamableHttp",
```

Comment:
I've seen this value more commonly as `"http"` or `"streamable-http"` but not `"streamableHttp"`. Which is right and is it the same universally across all clients?